spec/config/initializers/job_configurations_spec.rb

```diff
@@ -0,0 +1,13 @@
+# cron.each do |key, value|
+# [12] pry(main)*   value[:class].constantize.new
+# [12] pry(main)* end
+
+require 'rails_helper'
+
+describe 'job_configurations' do
+  it 'has valid cron jobs' do
```

Comment:
```suggestion
  it 'has valid cron jobs' do
    expect(Rails.application.config.good_job.cron).to be_present

```
let's make sure this isn't empty?